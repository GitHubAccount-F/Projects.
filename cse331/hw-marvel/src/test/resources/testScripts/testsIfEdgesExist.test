#Tests if edges properly form in marvel social network

#Creates graphs
LoadGraph graph1 noHerosInSameComicBook.csv
LoadGraph graph2 onlyTwoCharactersPerComic.csv
LoadGraph graph3 directPathInSameComic.csv
LoadGraph graph4 directPathNotInSameComics.csv

#Checks if edges were created
#Should produce no edges
ListChildren graph1 SpiderMan
ListChildren graph1 Wolverine
ListChildren graph1 Hulk

#Should have an edge both ways
ListChildren graph2 SpiderMan
ListChildren graph2 Venom

#Should produce a single edge
ListChildren graph2 Hulk
ListChildren graph4 IronMan
ListChildren graph4 Thor

#Should have many edges
ListChildren graph3 SpiderMan
ListChildren graph3 Carnage