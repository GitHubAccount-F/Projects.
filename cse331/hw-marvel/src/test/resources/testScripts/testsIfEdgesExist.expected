#Tests if edges properly form in marvel social network

#Creates graphs
loaded graph graph1
loaded graph graph2
loaded graph graph3
loaded graph graph4

#Checks if edges were created
#Should produce no edges
the children of SpiderMan in graph1 are:
the children of Wolverine in graph1 are:
the children of Hulk in graph1 are:

#Should have an edge both ways
the children of SpiderMan in graph2 are: Venom(Spider-Verse)
the children of Venom in graph2 are: SpiderMan(Spider-Verse)

#Should produce a single edge
the children of Hulk in graph2 are: Thor(Ragnarok)
the children of IronMan in graph4 are: Thor(Ragnarok) Venom(Stark-Tower-Falling)
the children of Thor in graph4 are: CaptainAmerica(Civil-War) IronMan(Ragnarok)

#Should have many edges
the children of SpiderMan in graph3 are: Carnage(Spider-Verse) MilesMorales(Spider-Verse) Venom(Spider-Verse)
the children of Carnage in graph3 are: MilesMorales(Spider-Verse) SpiderMan(Spider-Verse) Venom(Spider-Verse)